onmt/models/model_saver.py

```diff
@@ -117,6 +117,9 @@ def _save(self, step, model):
                 for key in keys_to_pop:
                     vocab[side].fields[0][1].vocab.stoi.pop(key, None)
 
+        # if hasattr(model, 'data_loader_step'):
```

Comment:
?

Reply:
The `data_loader_step` is something indent to iterate from where we had stopped. For now, I think this can be removed.

Reply:
Something similar to this https://github.com/OpenNMT/OpenNMT-py/pull/1826 ?

Reply:
yes.